src/Cli/dotnet/commands/dotnet-run/LocalizableStrings.resx

```diff
@@ -173,7 +173,7 @@ The current {1} is '{2}'.</value>
     <value>Arguments passed to the application that is being run.</value>
   </data>
   <data name="RunCommandExceptionCouldNotLocateALaunchSettingsFile" xml:space="preserve">
-    <value>The specified launch profile could not be located.</value>
+    <value>The specified launch profile '{0}' could not be located. Is it of the form ./Properties/launchSettings.json?</value>
```

Comment:
`Is it of the form ./Properties/launchSettings.json?` This should be removed as you can have a custom output path and I assume we don't expect people to use file-based launch profiles that often (VS is the interface between the two)